src/pages/settings/Subscription/SubscriptionDetails/index.tsx

```diff
@@ -44,33 +46,30 @@ function SubscriptionDetails() {
         setSelectedOption(option);
     };
 
-    // This section is only shown when the subscription is annual
-    // An onPress action is going to be assigned to these buttons in phase 2
-    let subscriptionSizeSection: React.JSX.Element | null = null;
+    const onSubscriptionSizePress = () => {
+        Navigation.navigate(ROUTES.SETTINGS_SUBSCRIPTION_SIZE);
+    };
 
-    if (privateSubscription?.type === CONST.SUBSCRIPTION.TYPE.ANNUAL) {
-        subscriptionSizeSection = privateSubscription?.userCount ? (
-            <MenuItemWithTopDescription
-                description={translate('subscription.details.subscriptionSize')}
-                title={`${privateSubscription?.userCount}`}
-                wrapperStyle={styles.sectionMenuItemTopDescription}
-                style={styles.mt5}
-            />
-        ) : (
+    // This section is only shown when the subscription is annual
```

Comment:
This section is visible even if the subscription type is not annual:

https://github.com/Expensify/App/assets/30054992/620d9741-67c1-4b03-bd33-a5312e7a106e

Reply:
Ah, I guess it does not show up if the set subscription is not annual.

Reply:
Condition was checking it against data coming straight from the API. Because change is only happening on FE now (API call will be implemented in the next phase) it was not working as expected. I've updated the condition to check it against the selectedOption that we store inside state of the component. It should work as expected now.